dev-python/coverage/coverage-7.2.2.recipe

```diff
@@ -0,0 +1,125 @@
+SUMMARY="A Python library for measuring code coverage"
+DESCRIPTION="coverage is tool for measuring code coverage of Python programs."
+HOMEPAGE="https://github.com/nedbat/coveragepy
+	https://coverage.readthedocs.io/
+	https://pypi.org/project/coverage/"
+COPYRIGHT="2001 Gareth Rees
+	2004-2023 Ned Batchelder"
+LICENSE="Apache v2"
+REVISION="1"
+pypiVersion="54/8a/db9d9cd24f96bb872eea151bb0d5c8cb6a96825b70a0cfaf07bceab2884d"
+SOURCE_URI="https://files.pythonhosted.org/packages/$pypiVersion/coverage-$portVersion.tar.gz"
+CHECKSUM_SHA256="36dd42da34fe94ed98c39887b86db9d06777b1c8f860520e21126a75507024f2"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	coverage$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+
+PYTHON_PACKAGES=(python39 python310)
+PYTHON_VERSIONS=(3.9 3.10)
+defaultVersion=3.9
+for i in "${!PYTHON_PACKAGES[@]}"; do
+	pythonPackage=${PYTHON_PACKAGES[i]}
+	pythonVersion=${PYTHON_VERSIONS[$i]}
+
+	eval "PROVIDES_$pythonPackage=\"
+		${portName}_$pythonPackage = $portVersion
+		cmd:coverage_$pythonVersion = $portVersion
+		\""
+
+	# Provide coverage and coverage3 for the default version
+	if [ "$pythonVersion" = "$defaultVersion" ]; then
+		eval "PROVIDES_${pythonPackage}+=\"
+			cmd:coverage = $portVersion
+			cmd:coverage${pythonVersion%.*} = $portVersion
+			\""
+	fi
+
+	if [ "$targetArchitecture" = "x86_gcc2" ]; then
+		eval "PROVIDES_${pythonPackage}+=\"
+			coverage_$pythonPackage = $portVersion
+			\""
+	fi
+
+	eval "REQUIRES_$pythonPackage=\"
+		haiku$secondaryArchSuffix
+		lib:libpython$pythonVersion$secondaryArchSuffix
+		\""
+	eval "BUILD_REQUIRES+=\"
+		setuptools_$pythonPackage
+		cmd:groff
+		cmd:rst2html.py
+		cmd:rst2man.py
```

Comment:
These should go in BUILD_PREREQUIRES. IMO they should be out of the loop. cmd:gcc$secondaryArchSuffix too.